src/OpenStack.Net/OpenStack/Services/ObjectStorage/V1/ObjectStorageServiceExtensions.cs

```diff
@@ -0,0 +1,842 @@
+﻿// Intentionally placed in the outer scope so IProgress<T> resolves to System.IProgress<T> instead of
```

Comment:
This workaround is no longer required (see a0b77be2cf1a3d5566a40f7af4509824a37db5a2).